simplified/cli.go

```diff
@@ -52,6 +53,15 @@ type CLIRunRequest struct {
 
 var errUnknownTool = errors.New("unknown tool")
 
+// Timeout for CLI command execution
+const toolCommandExecutionTimeout = 120 * time.Second
```

Comment:
```suggestion
const toolCommandExecutionTimeout = 9 * time.Minute
```
I suggest making this more inline to the max timeout of an extension call in general (10m).
That way a user can expect this to work the same way across the board and not have corner cases where timeouts are very different.

Reply:
I was doing some more testing and thinking and I'm starting to wonder if such a long timeout indeed makes sense.

I know it's a very conservative value, but if any of those CLI commands take more than a couple of minutes to finish, this likely indicates an unrecoverable issue from which we should abort early + provide clear indication to the user via error (e.g. missing arguments, network error, command is waiting for an input, command hangs or similar).

Even running `aws ec2 list-instances` with many instances or similar shouldn't take more than a couple of 10's of seconds.

So I'm kind back on this one - I think 2-3 minutes should be more than enough, but not a big deal.

Reply:
Counter example: running a cloud build
The issue is that we have not tested all the commands we have in our Cli wrapper to confirm their speed in real environments, and because this is extensible we don't know what other users will be using it to run.
I think the best thing we could do is give it a default that can be modified by the implementer.